### PR TITLE
Add/remove

### PR DESCRIPTION
**Pull request by CELYNEBIRUNGI**
Added  Css file
Updated Html  fIle 
Modified Readme file by including Celyne as an arthur.
Fixed Linter errors

### CODE REVIEW

**Comment by mailsg** [comment]
Hi,

1. This PR is raised to showcase the implementation of JavaScript with a plain UI. 
2. Created a collection that keeps a list of books.
3. Created a function to add a new book to the collection, with title and author.
4. Created a function to remove a book from the collection
5. Display all books saved in the collection in the top part of the page.
6. Add button to add a new book. A new book is added to the collection. The new book is displayed in the page.
7. Remove button to remove the book from the collection. The correct book is removed from the collection. The correct book disappears from the page.
8. Data is preserved in the browser's memory by using `localStorage`.

File: script.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  });
+
+  document.querySelector('#addbookbutton').addEventListener('click', () => {
+    const bookobject = new Books();
+    if (document.getElementById('titlename').value !== '' && document.getElementById('authorname').value !== '') {
+      bookobject.title = document.getElementById('titlename').value;
+      bookobject.author = document.getElementById('authorname').value;
+      bookdata.push(bookobject);
+    }
+  });
```
Comment (Ismailco) [inline script.js]: - [ ] Your code should include a function to add a new book to the list. Kindly make sure to separate your code, you can create a function to add new books and then use it inside the event listener

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @CELYNEBIRUNGI, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — APPROVED** [review]
Hi @CELYNEBIRUNGI and @mailsg  👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Implemented functions to add and remove books. ✔️ 

## Optional suggestions

**- [optional] Please if you don't have a live demo and presentation at the moment, kindly remove them from your README file to make it more professional. Thanks. 🙏** 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mailsg** [comment]
@OyePriscilla, thank you for the approval, however kindly note that this project is a joint effort of myself @mailsg and @CELYNEBIRUNGI. You have tagged incorrectly.

**Comment by OyePriscilla** [comment]
> @OyePriscilla, thank you for the approval, however kindly note that this project is a joint effort of myself @mailsg and @CELYNEBIRUNGI. You have tagged incorrectly.
@CELYNEBIRUNGI so sorry for the mix-up, I had made necessary adjustment.

**Pull request merged**